### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeanniefinks**
Added minor edits and YOLACT citation as requested by the author

### CODE REVIEW

File: examples/dbolya-yolact/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ run the following examples.
 
 | Model Name     |      Stub      | Description |
 |----------|-------------|-------------|
-| yolact-base |  | Dense full precision YOLACT model with darknet53 backbone|
+| yolact-base |  | Dense full precision YOLACT model with a DarkNet-53 backbone|
```
Comment (jeanniefinks) [inline examples/dbolya-yolact/README.md:45]: @markurtz or @rahul-tuli should this have a stub format example in the blank column?

File: examples/dbolya-yolact/README.md [hunk 2]
```diff
@@ -76,3 +76,11 @@ python eval.py \
 
 Note: for quantized performance, your CPU must support VNNI instructions.
 Review `/proc/cpuinfo` for the flag `avx512_vnni` to verify chipset compatibility.
+
+## Citation
+@inproceedings{yolact-iccv2019,
```
Comment (jeanniefinks) [inline examples/dbolya-yolact/README.md:81]:
```suggestion
```bibtex
@inproceedings{yolact-iccv2019,
```

File: examples/dbolya-yolact/README.md [hunk 3]
```diff
@@ -76,3 +76,11 @@ python eval.py \
...
+  author    = {Daniel Bolya and Chong Zhou and Fanyi Xiao and Yong Jae Lee},
+  title     = {YOLACT: {Real-time} Instance Segmentation},
+  booktitle = {ICCV},
+  year      = {2019},
+}
```
Comment (jeanniefinks) [inline examples/dbolya-yolact/README.md:86]:
```suggestion
}
```

```

File: examples/dbolya-yolact/README.md (hunk 1 above)
Comment (rahul-tuli) [reply examples/dbolya-yolact/README.md:45]: Will update this as soon as model is uploaded to SparseZoo

**Review by markurtz — APPROVED** [review]

File: examples/dbolya-yolact/README.md (hunk 1 above)
Comment (markurtz) [reply examples/dbolya-yolact/README.md]: updated with the proper stubs and table. Going to land this as and we can open up another PR if there are additional issues

**Pull request merged**